jwst/datamodels/model_base.py

```diff
@@ -338,7 +338,12 @@ def close(self):
 
             for fd in self._files_to_close:
                 if fd is not None:
-                    log_with_traceback(f'{__name__}: {self}: Closing {fd}')
```

Comment:
Another option perhaps is to wrap this in `if sys.meta_path is not None:` and then change the log level to debug?

Reply:
Did consider an in-code fix, but felt that debugging messages at this level is a bit deep anyways. Debugging at this point is to actually figure out problems in other places, so hopefully not needed often and would crowd out other more useful messages.